src/target/riscv/riscv-013.c

```diff
@@ -552,6 +553,12 @@ static bool check_dbgbase_exists(struct target *target)
 			LOG_TARGET_ERROR(target, "Reached the end of DM chain (detected %u DMs in total).", count);
 			break;
 		}
+		if (next_dm >> info->abits) {
+			LOG_TARGET_ERROR(target, "The address of the next Debug Module does not fit into %u bits,"
+					"which is the width of the DMI bus address. This is a HW bug",
```

Comment:
```suggestion
			LOG_TARGET_ERROR(target, "The address of the next Debug Module does not fit into %u bits, "
					"which is the width of the DMI bus address. This is a HW bug",
```
Detail: There is a missing space in the message.

Reply:
Nice catch! Addressed

Reply:
thanks